Lab 1/Lab1/Validation/ValidatorInputField.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Lab1.LogsAndExceptions;
+
+namespace Lab1
```

Comment:
Just a reminder about namespaces.
Namespaces should be the same as the folders structure. Fix in all places.